### PR TITLE
fix(combobox): placeholder only appears when there is no value on combobox

### PR DESCRIPTION
**Pull request by andresin87**
**TASK**: 

### Types of changes

- [x] 🪲 Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: packages/components/combobox/src/ComboboxInput.tsx [hunk 1]
```diff
@@ -75,6 +75,10 @@ export const Input = forwardRef(
       ref: inputRef,
     })
 
+    const { selectedItems, selectedItem, multiple } = ctx
```
Comment (Powerplex) [inline packages/components/combobox/src/ComboboxInput.tsx:78]: I favor if we don't destructure from `ctx`, especially if it's only used once. As the code get more complex, it helps see at first glance what comes from the context vs the props.

File: packages/components/combobox/src/ComboboxInput.tsx [hunk 2]
```diff
@@ -75,6 +75,10 @@ export const Input = forwardRef(
...
+
+    const hasPlaceholder = multiple ? selectedItems.length === 0 : selectedItem === null
```
Comment (andresin87) [inline packages/components/combobox/src/ComboboxInput.tsx:80]:
```suggestion
    const hasPlaceholder = ctx.multiple ? ctx.selectedItems.length === 0 : ctx.selectedItem === null
```

Comment (andresin87) [reply packages/components/combobox/src/ComboboxInput.tsx:80]: @Powerplex ?

**Review by Powerplex — APPROVED** [review]

**Comment by elisa-hery** [comment]
LGTM, @adevinta/spark-design do you want to check it ?

**Comment by JeremiasUX** [comment]
Yes, looks good to me too. Thanks :)

**Pull request merged**